pyspecProcScripts/first_level/QESR_rescale.py

```diff
@@ -16,35 +78,82 @@
+                        proportionality constant in your pyspecdata config
+                        file.  Typically this is one value that doesn't need
+                        changing
+    diameter_name:      str
+                        The key corresponding to the diameter of the capillary
+                        tube used in the ESR experiment. This enables us to
+                        calculate a reliable concentration regardless of which
+                        capillary was used.
+    Returns
+    =======
+    Denominator of equation 2-17 divided by the proportionality constant. Also
+    includes a factor of 1e-6 to convert Molar to micromolar.
+    """
+    calibcache.use_calibration(calibration_name)
+    calibcache.use_diameter(diameter_name)
     # {{{ determine the signal denominator from the parameters of interest
     G_R = Q_(*d.get_prop("Gain"))
     C_t = Q_(*d.get_prop("ConvTime"))
     power = Q_(*d.get_prop("Power"))
     B_m = Q_(*d.get_prop("ModAmp"))
-    Q = Q_(default_Q, "dimensionless")  # hard set Q value
+    Q = Q_(calibcache.default_Q, "dimensionless")  # hard set Q value
+    d = Q_(calibcache.d, "mm")  # hard set diameter
     n_B = Q_(1, "dimensionless")  # calculate this
     S = Q_(0.5, "dimensionless")
     c = Q_(
         1, "dimensionless"
-    )  # the first fraction on pg 2-17 -- essentially the conversion factor
-    signal_denom = G_R * C_t * sqrt(power) * B_m * n_B * S * (S + 1) * Q
-    signal_denom = signal_denom.to(Q_("G") * sqrt(Q_("W")) * Q_("s"))
+    )  # the first fraction on eq 2-17 -- in bruker E500 manual
+    signal_denom = (
+        G_R * C_t * sqrt(power) * B_m * n_B * S * (S + 1) * Q * d**2
+    )
+    signal_denom = signal_denom.to(
+        Q_("G") * sqrt(Q_("W")) * Q_("s") * Q_("m") ** 2
+    )
     # }}}
-    logging.debug(
+    logger.debug(
         strm(
-            f"$G_R={G_R:~P}$\n",
-            f"$C_t={C_t:~P}$\n",
-            f"$power={power:~P}$\n",
-            f"$B_m={B_m:~P}$\n",
-            f"$Q={Q:~P} $\n",
-            f"$n_B={n_B:~P} $\n",
-            f"$S={S:~P} $\n",
-            f"$c={c:~P} $\n",
-            f"signal denom$={signal_denom:~P}$",
-            f"doubleint conversion$={dint_conversion}$",
+            f"$G_R={G_R:~L}$\n"
+            + f"$C_t={C_t:~L}$\n"
+            + f"$power={power:~L}$\n"
+            + f"$B_m={B_m:~L}$\n"
+            + f"$Q={Q:~L} $\n"
+            + f"$n_B={n_B:~L} $\n"
+            + f"$S={S:~L} $\n"
+            + f"$c={c:~L} $\n"
+            + f"$d={d:~L} $\n"
+            + f"signal denom$={signal_denom:~L}$"
+            + f"doubleint propFactor$={calibcache.dint_propFactor}$"
         )
     )
-    # normally, we divide by signal_denom.magnitude and multiply by dint_conversion and divide by 1e-6
-    return signal_denom.magnitude / dint_conversion * 1e-6
+    # normally, we divide by signal_denom.magnitude and
+    # multiply by calibcache.dint_propFactor and
+    # multiply by 1e-6
```

Comment:
don't change the words (you changed the last divide to multiply)?

Reply:
[here](https://github.com/jmfrancklab/proc_scripts/pull/106#discussion_r1688015474) you pointed out that the math is multiplying by 1e-6 so changed comment to match